apps/smartling/lambda/package.json

```diff
@@ -4,17 +4,12 @@
   "private": true,
   "scripts": {
     "start": "LOCAL_DEV=true ts-node src/index.ts",
-    "clean": "rimraf built",
-    "build-lambda": "npm run clean && tsc",
-    "build-frontend": "cd ../frontend && npm run build",
-    "build": "npm run clean && npm run build-lambda && npm run build-frontend",
+    "build": "rimraf built && tsc",
     "lint": "tslint --project ./tsconfig.json",
-    "lint-fix": "tslint --project ./tsconfig.json --fix",
     "pretty": "prettier --config ./.prettierrc --write '{src/*.ts,src/*.spec.ts}'",
-    "test": "LOCAL_DEV=true jest **/*.spec.ts",
-    "test:ci": "npm run build-frontend && npm run test",
-    "test-dev": "LOCAL_DEV=true jest **/*.spec.ts --watch",
```

Comment:
`LOCAL_DEV=true` may be needed for local testing. A few things have changed here so maybe it is outdated.

Reply:
We only use it here: https://github.com/contentful/apps/blob/master/apps/smartling/lambda/src/index.ts#L119-L122 to treat app file as a server.

Tests still work w/o that.